src/SDKs/Azure.Base/data-plane/Azure.Base/Http/Pipeline/LoggingPolicy.cs

```diff
@@ -13,25 +13,27 @@ public class LoggingPolicy : HttpPipelinePolicy
         static readonly long s_delayWarningThreshold = 3000; // 3000ms
         static readonly long s_frequency = Stopwatch.Frequency;
 
-        int[] _excludeErrors = Array.Empty<int>();
+        private readonly HttpPipelinePolicy _next;
+        private readonly int[] _excludeCodes;
 
-        public readonly static LoggingPolicy Shared = new LoggingPolicy();
-
-        public LoggingPolicy(params int[] excludeErrors)
-            => _excludeErrors = excludeErrors;
+        public LoggingPolicy(HttpPipelinePolicy next, int[] excludeCodes)
```

Comment:
should we call these explicitly StatusCodes everywhere? as opposed to shortening to "codes"

Reply:
Maybe we should but it's outside of the scope of this PR.

Reply:
sounds good.